parity/cli/mod.rs

```diff
@@ -712,6 +712,10 @@ usage! {
 			"--tx-queue-strategy=[S]",
 			"Prioritization strategy used to order transactions in the queue. S may be: gas_price - Prioritize txs with high gas price",
 
+			ARG arg_tx_queue_allow_unknown_local: (bool) = true, or |c: &Config| c.mining.as_ref()?.tx_queue_allow_unknown_local.clone(),
```

Comment:
Maybe change this to a flag, as it is a boolean?

Reply:
I wanted the default (for the flag/art) to be true, which I can't do with a flag (seems like default must be false?) Also, unknown_local was the simplest name I could come up with that also explains the concept, and keeps all the flags/config items the same name (not negations). Length was also a cosideration. If there's a better name happy to change it.

(Like: allow_unknown_local is better than disallow_unknown_local (or deny_...), less thinking to do about what it means. and unknown_local is not great to start with, so already takes some thinking. Not super happy with the name, tbh)

Reply:
We have many "negative" flags already in cli, like `--no-warp`, `--no-discovery`, etc. And I searched the code, we don't have any boolean arg. So I still think that flag may be better for consistency, like `--disallow-unknown-local`/`--demote-unkown-local-transactions` or something like that.

Or maybe [like you said](https://github.com/paritytech/parity/pull/8882#issuecomment-396882476), we can set `--allow-unknown-local` default to false? I can't think of any case where this may break existing usages -- `eth_submitRawTransaction` specification doesn't require giving special promotion to the submitted transaction.